symfony/plugins/orangehrmTimePlugin/entity/Project.php

```diff
@@ -0,0 +1,175 @@
+<?php
+/**
+ * OrangeHRM is a comprehensive Human Resource Management (HRM) System that captures
+ * all the essential functionalities required for any enterprise.
+ * Copyright (C) 2006 OrangeHRM Inc., http://www.orangehrm.com
+ *
+ * OrangeHRM is free software; you can redistribute it and/or modify it under the terms of
+ * the GNU General Public License as published by the Free Software Foundation; either
+ * version 2 of the License, or (at your option) any later version.
+ *
+ * OrangeHRM is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY;
+ * without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+ * See the GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License along with this program;
+ * if not, write to the Free Software Foundation, Inc., 51 Franklin Street, Fifth Floor,
+ * Boston, MA  02110-1301, USA
+ */
+
+namespace OrangeHRM\Entity;
+
+use Doctrine\Common\Collections\ArrayCollection;
+use Doctrine\Common\Collections\Collection;
+use Doctrine\ORM\Mapping as ORM;
+use OrangeHRM\Entity\Decorator\DecoratorTrait;
+use OrangeHRM\Entity\Decorator\ProjectDecorator;
+
+/**
+ * @method ProjectDecorator getDecorator()
+ * @ORM\Table(name="ohrm_project")
+ * @ORM\Entity
+ *
+ */
+class Project
+{
+    use DecoratorTrait;
+
+    /**
+     * @var int
+     *
+     * @ORM\Column(name="project_id", type="integer", length=13)
+     * @ORM\Id
+     * @ORM\GeneratedValue(strategy="AUTO")
+     *
+     */
+    private int $id;
+
+    /**
+     * @var Customer
+     * @ORM\ManyToOne(targetEntity="OrangeHRM\Entity\Customer", inversedBy="project", cascade={"persist", "remove"})
+     * @ORM\JoinColumn(name="customer_id", referencedColumnName="customer_id",nullable=false)
+     */
+    private Customer $customer;
+
+    /**
+     * @var string|null
+     * @ORM\Column(name="name",type="string",length=100)
+     */
```

Comment:
```suggestion
    /**
     * @var string|null
     *
     * @ORM\Column(name="name", type="string", length=100)
     */
```
preferred this pattern for all other attribute annotations. (proper spacing)